docs/codeql/support/reusables/frameworks.rst

```diff
@@ -171,6 +171,8 @@ Python built-in support
    Twisted, Web framework
    Flask-Admin, Web framework
    starlette, Asynchronous Server Gateway Interface (ASGI)
+   ldap, Lightweight Directory Access Protocol (LDAP)
```

Comment:
Since we've listed `PyYAML` and not `yaml`, I guess we should continue listing the PyPI package name instead of the name used for importing in the Python code. We could probably do better to highlight what this is expected to be though :grimacing: 

```suggestion
   python-ldap, Lightweight Directory Access Protocol (LDAP)
```